### ISSUE TITLE
Probably everything. We need to review. We'll update.

### ISSUE THREAD

**Issue by DavidGmidd** [ProfessorAmanda/econsimulations#8]

**Comment by ProfessorAmanda** [comment]
Yes. Everything! So far we only have a sketch of this one constructed. Once we are ready to work on it full-time, I will send a more detailed description of what we hope to achieve.

**Comment by ProfessorAmanda** [comment]
Going to close this in favor of more specific issues.
